packages/mobile/src/screens/profile-screen/ProfileInfo.tsx

```diff
@@ -98,8 +107,25 @@ export const ProfileInfo = (props: ProfileInfoProps) => {
   const isReachable = useSelector(getIsReachable)
   const accountHandle = useSelector(getUserHandle)
   const styles = useStyles()
+  const dispatch = useDispatch()
   const { isEnabled: isChatEnabled } = useFeatureFlag(FeatureFlags.CHAT_ENABLED)
 
+  const profileUserId = useSelector((state) =>
+    getProfileUserId(state, params.handle)
+  )
+  const { canChat } = useSelector((state) => getCanChat(state, profileUserId))
+
+  useEffect(() => {
+    dispatch(fetchBlockees())
+    dispatch(fetchBlockers())
+  }, [dispatch, params, profileUserId])
```

Comment:
nit: I know this is merged already but probably worth a comment about these extra dependencies